dotCMS/src/main/java/com/dotcms/concurrent/DotConcurrentFactory.java

```diff
@@ -269,6 +275,28 @@ public DotSubmitter getSubmitter () {
         return this.getSubmitter(DOT_SYSTEM_THREAD_POOL);
     } // getSubmitter.
 
+    /**
+     * Register a submitter creator and gets the submitter
+     * @param name    {@link String}
+     * @param creator {@link SubmitterConfigCreator}
+     * @return DotSubmitter
+     */
+    public DotSubmitter getSubmitter (final String name, final SubmitterConfigCreator creator) {
```

Comment:
I would rename `SubmitterConfigCreator` to `SubmitterConfig` and `SubmitterConfigCreatorBuilder` to `SubmitterConfigBuilder`